samples/led/main.c

```diff
@@ -1,19 +1,13 @@
+#include <hal/led.h>
 #include <hal/xbox.h>
-#include <xboxkrnl/xboxkrnl.h>
-
-#define SMC_REG_LEDMODE             0x07
-#define SMC_REG_LEDSEQ              0x08
 
 void main(void)
 {
-    // Set LED to be user controlled
-    HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
-
     // LED pattern is:
-    //   Red:   0xC = 1100 (ON,  ON,  OFF, OFF;   ON,  ON,  OFF, OFF;   ...)
-    //   Green: 0x0 = 0000 (OFF, OFF, OFF, OFF;   OFF, OFF, OFF, OFF;   ...)
+    //   Red:   (ON,  ON,  OFF, OFF;   ON,  ON,  OFF, OFF;   ...)
+    //   Green: (OFF, OFF, OFF, OFF;   OFF, OFF, OFF, OFF;   ...)
```

Comment:
These comments should be removed.

They serve no purpose here.

The comments could be moved into the LED API code, but they are of such poor quality that they are probably more confusing than doing any good.

The new API is pretty descriptive; the only comment that's necessary is that each of the states, from left-to-right, will be shown for X ms and then it will repeat after 4*X ms.